backend/open_webui/internal/pocketbase/FILES_INTEGRATION.md

```diff
@@ -0,0 +1,44 @@
+# Files Integration with PocketBase
```

Comment:
Reference to https://pocketbase.io/docs/files-handling/ in this file, for reference. Also, in other design docs, reference to the relevant pages within https://pocketbase.io/docs/.